### PR TITLE
CM_Db_Statement::execute: remove variables from exception message

### PR DESCRIPTION
**Pull request by fvovan**

### CODE REVIEW

**Comment by fvovan** [comment]
@njam please review

File: library/CM/Db/Client.php [hunk 1]
```diff
@@ -191,7 +191,10 @@ public function createPdoStatement($sqlTemplate) {
                     $this->connect();
                     continue;
                 }
-                throw new CM_Db_Exception('Cannot prepare statement (retried ' . $try . 'x): ' . $e->getMessage());
+                throw new CM_Db_Exception('Cannot prepare statement', null, [
```
Comment (njam) [inline library/CM/Db/Client.php]: Let's add "database" to the message. How about "Cannot prepare SQL statement"?

File: library/CM/Db/Statement.php [hunk 2]
```diff
@@ -44,8 +44,11 @@ public function execute(array $parameters = null, $disableQueryBuffering = null)
                     $this->_reCreatePdoStatement();
                     continue;
                 }
-                throw new CM_Db_Exception('Cannot execute statement (retried ' . $try . 'x): ' . $e->getMessage() .
-                    ' (query: `' . $this->_pdoStatement->queryString . '`)');
+                throw new CM_Db_Exception('Cannot execute statement', null, [
```
Comment (njam) [inline library/CM/Db/Statement.php]: "Cannot execute SQL statement"?

**Comment by fvovan** [comment]
@njam please review

**Pull request merged**